apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -128,21 +131,26 @@ function ValidatorPageResult(): JSX.Element {
 
     const tableColumns = useMemo(() => {
         if (!data || !validatorEvents) return null;
+        const includeColumns = [
+            'Name',
+            'Stake',
+            'APY',
+            'Commission',
+            'Last Epoch Rewards',
+            'Voting Power',
+            'Status',
+        ];
+
+        if (!isFixedGasPrice) {
+            includeColumns.push('Proposed next Epoch gas price');
+        }
+
         return generateValidatorsTableColumns({
```

Comment:
wdyt about the code inside this function in https://github.com/iotaledger/iota/blob/1be7d547a139911f281df828a70fcea595906147/apps/explorer/src/lib/ui/utils/generateValidatorsTableColumns.tsx#L148-L149 ? Should we try to clean it up also so when we remove the feature we leave no debris behind?

Reply:
Removal task was updated with a reminder https://github.com/iotaledger/iota/issues/5882